ingest/ingest_files.py

```diff
@@ -106,6 +131,7 @@ def get_next_line(self, *, increase_line_count=True, split_line=True):
         """Returns a single line of txt, csv or tsv files"""
 
         next_row = next(self.file)
+        print(next_row)
```

Comment:
Another print statement here that seems superfluous.

Reply:
It is. 6f04851